.travis.yml

```diff
@@ -41,3 +41,5 @@ jobs:
         - BEHAT_TEST=1
         - REQUIRE_GRAPHQL="^4@dev"
         - BEHAT_SUITE="cms"
+    - env:
+        - NPM_TEST=1
```

Comment:
instead of this, could you update the provision to standard-jobs-fixed-npm.yml?  That'll add in an npm job using the shared config